api/command/mixins/team.py

```diff
@@ -0,0 +1,701 @@
+            team.discard_team_lead(rem_user.github_id)
+
+        removed = cast(bool, self._db_facade.store(team))
+        if removed:
+            logging.info(f"Member successfully removed: {team.__str__()}")
+        else:
+            logging.error("Member remove unsuccessful")
+        return removed
+
+    def team_edit(self,
+                  user_id: str,
+                  gh_team_name: str,
+                  display_name: str = None,
+                  platform: str = None) -> bool:
+        """
+        Edit the properties of a specific team.
+
+        Team leads can only edit the teams that they are a part of, but admins
+        can edit any team.
+
+        :param user_id: Slack ID of user who called command
+        :param display_name: display name to change to if not None
+        :param platform: platform to change to if not None
+        :raises: LookupError if the calling user or team to edit
+                 could not be found
+        :raises: RuntimeError if more than one team has the specified
+                 team name
+        :raises: PermissionError if the calling user does not have sufficient
+                 permissions to edit the specified team
+        :return: True if the edit was successful, False otherwise
+        """
+        logging.info("Team edit command API called")
+        try:
+            command_user = self._db_facade.retrieve(User, user_id)
+            logging.debug(f"Calling user: {command_user.__str__()}")
+        except LookupError:
+            msg = f"Calling user with Slack ID {user_id} not found!"
+            logging.error(msg)
+            raise LookupError(msg)
+
+        teams = self._db_facade.query(Team,
+                                      [('github_team_name', gh_team_name)])
+        num_teams = len(teams)
+
+        if num_teams < 1:
+            msg = f"No teams found with team name {gh_team_name}"
+            logging.error(msg)
+            raise LookupError(msg)
+        elif num_teams > 1:
+            msg = f"{num_teams} found with team name {gh_team_name}"
+            logging.error(msg)
+            raise RuntimeError(msg)
+        else:
+            team = teams[0]
+            logging.info(f"Team queried with team name {gh_team_name}:"
+                         f" {team.__str__()}")
+
+        if not check_permissions(command_user, team):
+            msg = f"User with ID {user_id} has insufficient permissions to " \
+                f"edit team {gh_team_name}"
+            logging.error(msg)
+            raise PermissionError(msg)
+
+        if display_name is not None:
+            logging.debug(f"Attaching display name {display_name} "
+                          f"to {gh_team_name}")
+            team.display_name = display_name
+
+        if platform is not None:
+            logging.debug(f"Attaching platform {platform} to {gh_team_name}")
+            team.platform = platform
+
+        edited = cast(bool, self._db_facade.store(team))

```

Comment:
See the first comment (I think its the first), which talks about specifying the type so that we don't have to cast.